### PR TITLE
data: OGEmanuel

### PR DESCRIPTION
**Pull request by OGEmanuel**

### CODE REVIEW

**Review by karnatisrinivas — COMMENTED** [review]
Looks good 👍

File: data/OGEmanuel.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/emmanuel-ogunmola-a5007a1a6/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Gmail",
+      "url": "emanuolumide@gmail.com",
```
Comment (Panquesito7) [inline data/OGEmanuel.json:25]: This link won't work. `mailto` must be used for this to work.
```suggestion
      "url": "mailto:emanuolumide@gmail.com",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Just the requested change to your email url then this should be good to merge. Let us know if you need any help.

**Comment by EmmaDawsonDev** [comment]
Closed as duplicate

**Pull request closed without merging**